### DISCUSSION TITLE
Surface Book 3: Problem with screen detaching

### DISCUSSION THREAD

**Discussion by Parz3val83** [linux-surface/linux-surface#1308] [Q&A]
Hello all, 
I recently installed DAT Linux on my surface book. It is currently the only OS I'm running, I did not set up a dual boot. I tried installing the surface linux kernel yesterday, but I got stuck on the black screen windows logo so I reinstalled linux frrom my usb. My system runs pretty well now, but If I move the laptop the screen slightly will register as disconnected and sometimes I have to hard shut down with the power button to get it working again. On windows if would register as detached for a few seconds then return to normal. I am new to linux and I am hesitant to try installing the kernel again. If anyone has any tips please advise.

**Comment by qzed** [comment]
I assume your SB has a discrete GPU in the base? If that is the case, it likely locks up because it doesn't handle being removed well when the driver is loaded. What you can do is to block the driver from loading (i.e. blacklist it). See [here](https://github.com/linux-surface/surface-hotplug/wiki/Runtime-Power-Management#configuring-the-nvidia-driver) for details. That should then behave just like on Windows.

Apart from that, I would suggest that you can try cleaning the contacts of the connector between base and tablet, if you have not done that already.
